sections/advanced/server-side-rendering.js

```diff
@@ -48,14 +48,16 @@ const ServerSideRendering = () => md`
 
   > \`sheet.getStyleTags()\` and \`sheet.getStyleElement()\` can only be called after your element is rendered. As a result, components from \`sheet.getStyleElement()\` cannot be combined with \`<YourApp />\` into a larger component.
 
+  You might need to install and use our babel plugin with its `ssr` option turned on.
```

Comment:
Given that our overall philosophy will shift in the next major toward a strong recommendation to use the babel plugin, I think it might make sense to change the working from "might" to "should". Thoughts @kitten?

Reply:
Seems reasonable :+1: